app/concerns/orderable.rb

```diff
@@ -20,12 +20,21 @@ def reorder_others(is_after)
       other_where = Hash[ordered_attributes.map do |atr|
         [atr, send(is_after ? "#{atr}_before_last_save" : "#{atr}_was")]
       end]
-      others = self.class.where(other_where).order('section_order asc')
+      if self.is_a?(Reply)
```

Comment:
This would be better if we just implemented .ordered on the other Orderable classes? Or we could define it using the concern to default to 'section_order asc' and then override it for Reply. This special casing is Not Stylistic if we can avoid it.